### DISCUSSION TITLE
So What Shall We Do Next.

### DISCUSSION THREAD

**Discussion by ghost** [Slow-Scallion4183/Adventure-Colussus#12] [General]
Can We Add The Main Part Of The Game?

**Comment by Slow-Scallion4183** [comment]
yeah sure! what do you have in mind?

**Reply by ghost** [reply]
to make some misc stuff.

**Reply by Slow-Scallion4183** [reply]
such as? story, npcs, etc?

**Reply by ghost** [reply]
yes, im good with basic stuff so i can make dialogs. who does pixel art for characters?

**Reply by ghost** [reply]
hello?

**Reply by Slow-Scallion4183** [reply]
sounds good. we can make dialog for npcs + interactions (input) and the story. (Get object + get another object = fight boss or something) <---- just an idea

**Reply by ghost** [reply]
ok! ill do dialogs!

**Reply by Slow-Scallion4183** [reply]
with 'pixel art' are you reffering to the ascii art?

**Reply by Slow-Scallion4183** [reply]
because you can do that if you want. i usually just search up '{insert name} ascii art

**Reply by ghost** [reply]
ok

**Reply by ghost** [reply]
do u have discord?

**Reply by Slow-Scallion4183** [reply]
and then just check out 'ascii_art_functions.py' in the master / development branch to see how to program it

**Reply by Slow-Scallion4183** [reply]
yes i have discord

**Reply by ghost** [reply]
ok

**Reply by ghost** [reply]
lets chat there in private. for surprise for other devs!

**Reply by Slow-Scallion4183** [reply]
Pro____grammer3

**Reply by Slow-Scallion4183** [reply]
^ is my username

**Reply by ghost** [reply]
ok

**Reply by ghost** [reply]
wait my mom

**Reply by Slow-Scallion4183** [reply]
hahah ok

**Reply by Slow-Scallion4183** [reply]
discord tag: #8054

**Reply by Slow-Scallion4183** [reply]
if you give me ur username + tag ill add u...

**Reply by ghost** [reply]
wait. ill add u lol

**Reply by ghost** [reply]
sent req.
